client/src/utils/csv_parser.ts

```diff
@@ -0,0 +1,158 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {parse} from '../../../node_modules/csv-parse';
+
+
+// Island enums for error checking
+enum Island {
+    NORTH_ISLAND = 'North Island',
+    SOUTH_ISLAND = 'South Island'
+  }
+
+// Student object that can be accessed
```

Comment:
Could you add a `TODO:` comment to note that this type will be eventually be replaced with a DTO class as defined in `common`